cmd/ci-operator/main.go

```diff
@@ -398,8 +398,10 @@ func (o *options) Complete() error {
 	if err != nil {
 		return results.ForReason("loading_config").WithError(err).Errorf("failed to load configuration: %v", err)
 	}
+	if config.CanonicalGoRepository != nil {
+		o.jobSpec.Refs.PathAlias = *config.CanonicalGoRepository
```

Comment:
Before, you set the PathAlias to `o.configSpec.CanonicalGoRepository`. Now, it's `config.CanonicalGoRepository`. Why?